### PR TITLE
Set default region for both machines and volumes

### PR DESCRIPTION
**Pull request by chessydk**
It seems like for some fly users (me and also reported by @joshmedeski here: https://github.com/epicweb-dev/epic-stack/issues/68), that the default region for machines and volumes are not the same and a deploy will fail because machine and mounted volume needs to be in same region. Specifying them explicitly solves the problem. I picked "sjc" a bit randomly, it could just as well be another, but it will never fit all users anyway.

Documentation for "primary_region": https://fly.io/docs/reference/configuration/#primary-region

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,11 +199,13 @@ Prior to your first deployment, you'll need to do a few things:
...
   production environments. Run the following (feel free to change the GB size
-  based on your needs):
+  based on your needs and the region of your choice
+  (`https://fly.io/docs/reference/regions/`). NB. If you do change the region,
+  make sure you change the `primary_region` in fly.toml as well):
 
   ```sh
-  fly volumes create data --size 1 --app epic-stack-template
-  fly volumes create data --size 1 --app epic-stack-template-staging
+  fly volumes create data --region sjc --size 1  --app epic-stack-template
```
Comment (kentcdodds) [inline README.md:207]:
```suggestion
  fly volumes create data --region sjc --size 1 --app epic-stack-template
```

File: README.md [hunk 2]
```diff
@@ -199,11 +199,13 @@ Prior to your first deployment, you'll need to do a few things:
 
 - Create a persistent volume for the sqlite database for both your staging and
   production environments. Run the following (feel free to change the GB size
-  based on your needs):
+  based on your needs and the region of your choice
+  (`https://fly.io/docs/reference/regions/`). NB. If you do change the region,
```
Comment (kentcdodds) [inline README.md:203]: I don't know what "NB" means here, but I think it makes sense without it.

```suggestion
  (`https://fly.io/docs/reference/regions/`). If you do change the region,
```

**Review by kentcdodds — APPROVED** [review]
I think this is a good temporary solution. Hopefully later we can remove this.

File: README.md (hunk 2 above)
Comment (chessydk) [reply README.md]: Yep, works fine without. N.B. was "Abbreviation of Latin nota bene. Used before a piece of important information to make readers notice it."

**Pull request merged**